packages/xo-server-backup-reports/src/index.js

```diff
@@ -679,6 +679,11 @@ class BackupReportsXoPlugin {
           status: nagiosStatus,
           message: nagiosMarkdown,
         }),
+      xo.sendIcinga2Status !== undefined &&
+        xo.sendIcinga2Status({
+          status: nagiosStatus === 0 ? 'OK' : 'CRITICAL',
```

Comment:
I can change `_sendReport` (it's a private method):
 - instead of `nagiosStatus` a boolean `success`
- if `success` => `OK` else `CRITICAL`

What do you think ?

Reply:
Or i can put the job status in argument.

Reply:
`_sendReport` can takes the job status and translate it to the corresponding status.